scripts/create_changelog.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that automatically makes updates to changelog and authors
+using release_summary.md.
+
+This script should only be run after release_info.py script is run
+successfully.
+"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+
+import argparse
+import collections
+import datetime
+import getpass
+import os
+import re
+import sys
+
+import feconf
+import python_utils
+
+from . import common
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PY_GITHUB_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'PyGithub-1.43.7')
+sys.path.insert(0, _PY_GITHUB_PATH)
+
+# pylint: disable=wrong-import-position
+import github # isort:skip
+# pylint: enable=wrong-import-position
+
+GIT_CMD_TEMPLATE_CHECKOUT = 'git checkout -- %s'
+# This should match the indentation of span and li elements used
+# in developer_names section of
+# core/templates/dev/head/pages/about-page/about-page.directive.html.
+SPAN_INDENT = '              '
+LI_INDENT = '                '
+# This line should match the line after div element for developer_names in
+# about-page.directive.html.
+LINE_AFTER_Z_DEVELOPER_NAMES = (
+    '            <p translate="I18N_ABOUT_PAGE_developer_nameS_'
+    'TAB_TEXT_BOTTOM" translate-values="{listOfNames: $ctrl.listOfNames}">\n')
+ABOUT_PAGE_FILEPATH = (
+    'core/templates/dev/head/pages/about-page/about-page.directive.html')
+AUTHORS_FILEPATH = os.path.join('AUTHORS', '')
+CHANGELOG_FILEPATH = os.path.join('CHANGELOG', '')
+CONTRIBUTORS_FILEPATH = os.path.join('CONTRIBUTORS', '')
+
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument(
+    '--github_username', help=('Your GitHub username.'), type=str)
+
+
+def update_sorted_file(filepath, new_list, last_comment_line):
+    """Updates the files AUTHORS and CONTRIBUTORS with a sorted list of
+    new authors or contributors.
+
+    Args:
+        filepath: str. The path of the file to update.
+        new_list: list(str). The list of new authors or contributors to
+            add to the file.
+        last_comment_line: str. The content of the line of last comment in
```

Comment:
I think you can just make this a constant here and not pass it in, it's the same for both files. Throw an exception if it can't be found.

Reply:
Done.